frappe/website/utils.py

```diff
@@ -284,6 +284,9 @@ def extract_title(source, path):
 
 	if "<!-- title:" in source:
 		title = re.findall('<!-- title:([^>]*) -->', source)[0].strip()
+		if '{{' in title:
+			title = re.findall(r'\{{(.*?)\}}', title)[0].strip()
+			title = eval(title)
```

Comment:
This enables a serious security vulnerability, because `title` is exposed as a user input at some places.